dynamic-config/config-apollo/pom.xml

```diff
@@ -44,5 +44,9 @@
       <groupId>javax.validation</groupId>
       <artifactId>validation-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-jdk14</artifactId>
+    </dependency>
```

Comment:
Why you modifiy this? I think can be deleted.

Reply:
Sorry,It's my fault.Mybe my project is not full compilation .A few days ago,When I run test in config-apollo,I found it print nothing.There is no need to add this dependency,I had remove it in a new commit.